RELEASE/scripts/autoscend/auto_choice_adv.ash

```diff
@@ -107,6 +107,13 @@ void main(int choice, string page)
 				set_property("auto_beatenUpCount", get_property("auto_beatenUpCount").to_int() + 1);
 			}
 			break;
+		case 1061: // Heart of Madness(Madness Bakery Quest)
+			if(internalQuestStatus("questM25Armorer") <= 1) {
+				run_choice(1);
+			} else {
+				run_choice(5);

```

Comment:
Might want to grab a tart while we are here if possible. can go in a followup PR